scripts/build-es-shims-data/es-shims-features.js

```diff
@@ -52,7 +52,11 @@ module.exports = {
     "String.prototype methods / String.prototype.endsWith",
   "String.prototype.includes":
     "String.prototype methods / String.prototype.includes",
-  "String.prototype.matchAll": [],
+  "String.prototype.matchAll": {
+    features: [
+      'String.prototype.matchAll / basic functionality',
+    ],
+  },
```

Comment:
The `matchAll` polyfill from @es-shims [expliticly handles](https://github.com/es-shims/String.prototype.matchAll/blob/master/polyfill.js#L7-L9) the "throws on non-global regex" case so we should consider it.

Thus, since we would have to list every subcategory of `String.prototype.matchAll`, we can use the "shorthand" version:

```suggestion
  "String.prototype.matchAll": "String.prototype.matchAll",
```

Reply:
changed